src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum Colors {
+    RED,
+    BLUE,
+    YELLOW,
+    GREEN,
+    PURPLE;
+
+    @Override
+    public String toString() {
+        return name();
+    }

```

Comment:
actually you don't need to override toString() rather use the name() method where you need to